ByBitApiService/src/main/java/org/example/Client/ByBitTickersClient.java

```
@@ -0,0 +1,59 @@
+package org.example.Client;
+
+import org.example.Model.Order.OrderResponse;
+import org.example.Model.Ticker.TickerResponse;
+import org.springframework.http.HttpStatusCode;
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.core.publisher.Mono;
+
+
+public class ByBitTickersClient {
+
+        WebClient webClient;
+
+    public ByBitTickersClient(WebClient webClient) {
+        this.webClient = webClient;
+    }
+
+
+    public OrderResponse getOrderBook(String tickerCategory, String tickerSymbol){
+        return webClient.get().uri(uriBuilder -> uriBuilder
+                        .path("/orderbook")
+                        .queryParam("category", tickerCategory)
+                        .queryParam("symbol", tickerSymbol)
+                        .build())
+
+                .retrieve()
+                .onStatus(
+                        HttpStatusCode::is4xxClientError,
+                        error -> Mono.error(new RuntimeException("API not found")))
+
+                .onStatus(
+                        HttpStatusCode::is5xxServerError,
+                        error -> Mono.error(new RuntimeException("Server is not responding")))
```

I think it's better to return a server error code